gateway/buckets.go

```diff
@@ -72,7 +72,8 @@ func (g *Gateway) renderBucketPath(c *gin.Context, ctx context.Context, threadID
 		render404(c)
 		return
 	}
-	if strings.HasSuffix(pth, ".textile/config.yml") {
+	// @todo: Remove this private bucket handling when the thread ACL is done.
```

Comment:
All the gateway methods will now show 404 for buckets that are private. Eventually, the public gateway will require an access token in the URL, at which point we can remove this handling and also remove the magic skeleton key token that the gateway is holding.